pkg/infra/remotecache/redis_storage.go

```diff
@@ -71,7 +82,7 @@ func (s *redisStorage) Set(key string, val interface{}, expires time.Duration) e
 	if err != nil {
 		return err
 	}
-	status := s.c.SetEx(key, expires, string(value))
+	status := s.c.Set(key, string(value), expires)
```

Comment:
SetEx is gone from the newer version of the redis lib and replaced with Set with always takes an expire param.